playbook-website/Gemfile.lock

```diff
@@ -285,6 +285,7 @@ GEM
 PLATFORMS
   arm64-darwin-21
   arm64-darwin-22
+  x86_64-darwin-20
```

Comment:
@jasperfurniss this is definitely in Nitro as well, so we should probably be fine shipping it, will leave it to you to make a final call on this. Rest of the code is good to go!